### PR TITLE
RGB matrix effects definiton fix

### PR DESCRIPTION
**Pull request by fOmey**
RGB matrix effects were incorrectly defined in the first pull request and made it through.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — APPROVED** [review]

**Comment by fOmey** [comment]
Mistakenly deleted branch thinking changes were merged.

**Review by daskygit — CHANGES_REQUESTED** [review]
The default keymap now compiles oversize, you can either use `LTO_ENABLE = yes` or remove some of the enabled RGB effects.

File: keyboards/atlantis/ak81_ve/rules.mk [hunk 1]
```diff
@@ -7,16 +7,17 @@ BOOTLOADER = atmel-dfu
 # Build Options
 #   change yes to no to disable
 #
-BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
+BOOTMAGIC_ENABLE = yes      # Bootmagic Lite
```
Comment (daskygit) [inline keyboards/atlantis/ak81_ve/rules.mk:10]:
```suggestion
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
```

File: keyboards/atlantis/ak81_ve/rules.mk [hunk 2]
```diff
@@ -7,16 +7,17 @@ BOOTLOADER = atmel-dfu
...
 MOUSEKEY_ENABLE = yes       # Mouse keys
 EXTRAKEY_ENABLE = yes       # Audio control and System control
 CONSOLE_ENABLE = no         # Console for debug
 COMMAND_ENABLE = no         # Commands for debug and configuration
-NKRO_ENABLE = yes           # Enable N-Key Rollover
+NKRO_ENABLE = yes           # N-Key Rollover
```
Comment (daskygit) [inline keyboards/atlantis/ak81_ve/rules.mk:15]:
```suggestion
NKRO_ENABLE = yes           # Enable N-Key Rollover
```

File: keyboards/atlantis/ak81_ve/rules.mk [hunk 3]
```diff
@@ -7,16 +7,17 @@ BOOTLOADER = atmel-dfu
...
 BACKLIGHT_ENABLE = no       # Keyboard backlight functionality
 RGBLIGHT_ENABLE = no        # Keyboard RGB underglow
-RGB_MATRIX_ENABLE = yes		# Enable RGB matrix effects
-RGB_MATRIX_DRIVER = WS2812
+RGB_MATRIX_ENABLE = yes		# RGB matrix effects
+RGB_MATRIX_DRIVER = WS2812	# RGB matrix driver
+LTO_ENABLE = yes			# Link time optimization
```
Comment (daskygit) [inline keyboards/atlantis/ak81_ve/rules.mk:20]:
```suggestion
RGB_MATRIX_ENABLE = yes     # Enable RGB matrix
RGB_MATRIX_DRIVER = WS2812  # RGB matrix driver
LTO_ENABLE = yes            # Link time optimization
```

File: keyboards/atlantis/ak81_ve/rules.mk [hunk 4]
```diff
@@ -7,16 +7,17 @@ BOOTLOADER = atmel-dfu
...
 AUDIO_ENABLE = no           # Audio output
-DYNAMIC_MACRO_ENABLE = yes
-ENCODER_ENABLE = yes
\ No newline at end of file
+DYNAMIC_MACRO_ENABLE = yes	# Macro recording in realtime
+ENCODER_ENABLE = yes		# Encoder
```
Comment (daskygit) [inline keyboards/atlantis/ak81_ve/rules.mk:23]:
```suggestion
DYNAMIC_MACRO_ENABLE = yes  # Macro recording in realtime
ENCODER_ENABLE = yes        # Encoder
```

**Review by daskygit — CHANGES_REQUESTED** [review]
Just some small white space corrections and edits to keep comments consistent with template. https://github.com/qmk/qmk_firmware/blob/master/data/templates/avr/rules.mk

File: keyboards/atlantis/ak81_ve/rules.mk [hunk 5]
```diff
@@ -7,16 +7,17 @@ BOOTLOADER = atmel-dfu
...
+COMMAND_ENABLE = no			# Commands for debug and configuration
+NKRO_ENABLE = yes			# N-Key Rollover
+BACKLIGHT_ENABLE = no		# Keyboard backlight functionality
+RGBLIGHT_ENABLE = no		# Keyboard RGB underglow
+RGB_MATRIX_ENABLE = yes		# RGB matrix effects
+RGB_MATRIX_DRIVER = WS2812	# RGB matrix driver
+LTO_ENABLE = yes			# Link time optimization
+AUDIO_ENABLE = no			# Audio output
+DYNAMIC_MACRO_ENABLE = yes	# Macro recording in realtime
+ENCODER_ENABLE = yes		# Encoder
```
Comment (fauxpark) [inline keyboards/atlantis/ak81_ve/rules.mk:23]:
```suggestion
BOOTMAGIC_ENABLE = yes      # Bootmagic Lite
MOUSEKEY_ENABLE = yes       # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
NKRO_ENABLE = yes           # Enable N-Key Rollover
BACKLIGHT_ENABLE = no       # Keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Keyboard RGB underglow
AUDIO_ENABLE = no			# Audio output
RGB_MATRIX_ENABLE = yes
RGB_MATRIX_DRIVER = WS2812
LTO_ENABLE = yes
DYNAMIC_MACRO_ENABLE = yes
ENCODER_ENABLE = yes
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/atlantis/ak81_ve/rules.mk [hunk 6]
```diff
@@ -7,16 +7,17 @@ BOOTLOADER = atmel-dfu
...
+BOOTMAGIC_ENABLE = yes      # Bootmagic Lite
 MOUSEKEY_ENABLE = yes       # Mouse keys
 EXTRAKEY_ENABLE = yes       # Audio control and System control
 CONSOLE_ENABLE = no         # Console for debug
 COMMAND_ENABLE = no         # Commands for debug and configuration
 NKRO_ENABLE = yes           # Enable N-Key Rollover
 BACKLIGHT_ENABLE = no       # Keyboard backlight functionality
 RGBLIGHT_ENABLE = no        # Keyboard RGB underglow
-RGB_MATRIX_ENABLE = yes		# Enable RGB matrix effects
+AUDIO_ENABLE = no			# Audio output
```
Comment (fauxpark) [inline keyboards/atlantis/ak81_ve/rules.mk:18]:
```suggestion
AUDIO_ENABLE = no           # Audio output
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Review by zvecr — APPROVED** [review]

**Pull request merged**